vip-helpers/vip-utils.php

```diff
@@ -1058,8 +1058,16 @@ function wpcom_vip_load_plugin( $plugin = false, $folder = false, $load_release_
 
 		return _wpcom_vip_include_plugin( $includepath );
 	} else {
+		$error_msg = "Unable to load plugin '$plugin' using wpcom_vip_load_plugin()!";
 		if ( ! WPCOM_IS_VIP_ENV ) {
-			die( "Unable to load $plugin using wpcom_vip_load_plugin()!" );
+			die( $error_msg );
+		} else {
+			// On VIP we try to both notify the user...
+			trigger_error( $error_msg, E_USER_WARNING );
+			// ...And trigger a New Relic notice, if the extension is available
+			if ( extension_loaded( 'newrelic' ) && function_exists( 'newrelic_notice_error' ) ) {
```

Comment:
Aside: I wonder if there's value in abstracting this into a separate logging library / function (i.e. which calls both `trigger_error` and `newrelic_notice_error`)? Out of scope for this PR so no change needed, but might be good to open an issue and discuss what that could look like.

Reply:
https://github.com/Automattic/vip-go-mu-plugins/issues/913